contracts/SlashIndicator.sol

```diff
@@ -80,6 +80,7 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
 
   /*********************** External func ********************************/
   function slash(address validator) external onlyCoinbase onlyInit oncePerBlock onlyZeroGasPrice{
+    require(IBSCValidatorSet(VALIDATOR_CONTRACT_ADDR).isCurrentValidator(validator), "not current validator");
```

Comment:
Can we do not use require but simply return, since `slash` is called by consensus engine,  we don't want consensus engine fail in any case.

Reply:
fixed